build.sh

```diff
@@ -20,7 +20,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")"
 #generator="Unix Makefiles"
 
 type=${type:-"Debug"}
-generator=${generator:-"Ninja"}
+generator=${generator:-"Unix Makefiles"}
```

Comment:
Why is this change needed?

Reply:
Oh it isn't I need to change it locally since I didn't have Ninja I will fix that

Reply:
Oh it isn't I need to change it locally since I didn't have Ninja I will fix that